DigitalLearningSolutions.Web/Styles/trackingSystem/courseContent.scss

```diff
@@ -1,7 +1,28 @@
 ﻿@import "~nhsuk-frontend/packages/core/all";
+@import "../shared/searchableElements/common";
 
-dd.nhsuk-summary-list__value {
+.summary-list-right-align-tags {
   @include mq($from: tablet) {
     text-align: right;
   }
 }
+
+.select-all-button {
+  @extend .right-submit-button;
+  background-color: $button-dark-blue;
+  border-radius: 4px;
+  margin-right: nhsuk-spacing(2);
+}
+
+.deselect-all-button {
+  @extend .right-submit-button;
+  background-color: $button-dark-blue;
+  border-radius: 4px;
+  margin-right: nhsuk-spacing(2);
+}

```

Comment:
Changing to ids complicates the javascript since we have to add the same event listener to the distinct ids. I'd prefer to just make a common css class then both extend the common one.

Reply:
Could we do something like a shared class that applies the styles and separate classes to differentiate them in code then?